launcher/container_runner.go

```diff
@@ -334,11 +336,22 @@ func (r *ContainerRunner) refreshToken(ctx context.Context) (time.Duration, erro
 
 	// Get token expiration.
 	claims := &jwt.RegisteredClaims{}
-	_, _, err = jwt.NewParser().ParseUnverified(string(token), claims)
+	_, parts, err := jwt.NewParser().ParseUnverified(string(token), claims)
 	if err != nil {
 		return 0, fmt.Errorf("failed to parse token: %w", err)
 	}
 
+	// Print out the claims in the jwt payload
+	rawDecodedPayload, err := base64.RawStdEncoding.DecodeString(string(parts[1]))
+	if err != nil {
+		return 0, err
+	}
+	var jsonBuffer bytes.Buffer
+	if err := json.Indent(&jsonBuffer, []byte(rawDecodedPayload), "", "  "); err != nil {
+		return 0, err
+	}
```

Comment:
Then we could remove this code an just do:
```
claimsString, err := json.MarshalIndent(claims, "", "  ")
r.logger.Println(claimsString)
```

Reply:
Done